app/build.gradle.kts

```diff
@@ -7,12 +7,12 @@ plugins {
 
 android {
     namespace = "com.stripe.aod.sampleapp"
-    compileSdk = 33
+    compileSdk = 34
 
     defaultConfig {
         applicationId = "com.stripe.aod.sampleapp"
         minSdk = 28
-        targetSdk = 33
+        targetSdk = 34
```

Comment:
Just to confirm, was AoD verified to work on API 34 + `targetSdk 34`? We have had issues with some other reader types when using this: https://github.com/stripe/stripe-terminal-android/issues/387

I don't believe any of the things in that issue affecting BT/USB/TTPA discovery + connection should apply to handoff, but just want to make sure this was manually verified.

Reply:
Good call. I reverted this for now. Will test and bump independently.